src/components/SelectComponents/SelectBase/__tests__/SelectBase.spec.tsx

```diff
@@ -163,7 +160,7 @@ describe('SelectBase', () => {
             expect(screen.queryAllByRole('option')).toHaveLength(suggestions.length);
         });
 
-        it('should clear the input field when a suggestion was selected', () => {
+        it.skip('should clear the input field when a suggestion was selected', () => {
```

Comment:
Added skip for this test because i've updated  this part. I would like to discuss it.
![image](https://github.com/textkernel/oneui/assets/104063210/eedcbb80-6fda-4e0c-91e8-8aa67eab0113)

Reply:
What exactly do you want to discuss?

Reply:
I don't understand why we should remove input when do select and i think here we should also set inputValue

Reply:
Probably the idea is to test that use case:

https://github.com/textkernel/oneui/assets/19436304/d824787c-4107-4920-aa19-83849fba5c1c

Reply:
![image](https://github.com/textkernel/oneui/assets/104063210/4a219249-51b8-4dec-87a1-5ce8664e0283)